rmb-sdk-go/peer/peer.go

```diff
@@ -44,6 +44,15 @@ type peerCfg struct {
 	enableEncryption bool
 	encoder          encoder.Encoder
 	cacheFactory     cacheFactory
+	relayCooldown    time.Duration
+}
+
+// WithRelayCooldown sets the cooldown duration for relay failover and retry logic.
+// If not set, defaults to 10 seconds.
```

Comment:
I think 10 seconds is much? no?

Reply:
No at all. The cool down will affect only the order in which relays would be tried out